File: SmartValley.FrontEnd/src/app/api/estimates/responses/scoring-report-response.ts
import {ExpertResponse} from '../../expert/expert-response';
import {ScoringReportInAreaResponse} from './scoring-report-in-area-response';

export interface ScoringReportResponse {
  experts: Array<ExpertResponse>;
  scoringStatistics: Array<ScoringReportInAreaResponse>;
}

File: SmartValley.WebApi/Estimates/Responses/ScoringReportResponse.cs
using System.Collections.Generic;
using SmartValley.WebApi.Experts.Responses;

namespace SmartValley.WebApi.Estimates.Responses
{
    public class ScoringReportResponse
    {
        public IReadOnlyCollection<ExpertResponse> Experts { get; set; }
        
        public IReadOnlyCollection<ScoringReportInAreaResponse> ScoringReportsInArea { get; set; }
    }
}